opentreemap/treemap/js/src/buttonEnabler.js

```diff
@@ -68,7 +68,7 @@ function fullyDisable($el, disabledTitle) {
 
 exports.run = function (options) {
     var $elements = $(enablePermSelector),
-        loggedIn = options && options.config ? options.config.loggedIn : true;
+        config = options.config;
```

Comment:
Merged before seeing your comment. Not passing in `options` was a momentary bad idea of mine; all uses of `buttonEnabler` supply `options.config` so future users aren't likely to omit it.